cloud/gcp-functions/cmd/disks.go

```diff
@@ -0,0 +1,46 @@
+package disks
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"google.golang.org/api/compute/v1"
+)
+
+// CleanDisks : remeve unused disks
+func CleanDisks(http.ResponseWriter, *http.Request) {
+	ctx := context.Background()
+	computeService, err := compute.NewService(ctx)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Consider adding some context to the errors. E.g. log.Fatal("Error: create service: ",  err)

Reply:
done